src/RayMarchedAtlasVolume.ts

```diff
@@ -73,20 +88,22 @@ export default class RayMarchedAtlasVolume implements VolumeRenderImpl {
 
     this.geometryTransformNode.add(this.boxHelper, this.tickMarksMesh, this.geometryMesh);
 
-    this.updateVolumeDimensions();
     this.settings = settings;
     this.updateSettings(settings, SettingsFlags.ALL);
+    // TODO this is doing *more* redundant work! Fix?
+    this.updateVolumeDimensions();
   }
 
   public updateVolumeDimensions(): void {
     const { normPhysicalSize, normRegionSize } = this.volume;
     // Set offset
-    this.geometryMesh.position.copy(this.volume.getContentCenter());
+    this.geometryMesh.position.copy(this.volume.getContentCenter().multiply(this.settings.scale));
```

Comment:
Why does a volume's scale affect its offset position? Is the scaling origin always the origin of the scene?

Reply:
The offset position is relative to the center of the volume, which for this viewer should almost always be the origin of the scene. It's there to compensate for an irregular subregion. E.g. if the subregion in the x-axis were 0.5 - 1.0, `getContentCenter` would return 0.25 in the x-axis, because the center of the mesh has to move from the center of the volume bounding box to halfway to its edge:
![image](https://github.com/user-attachments/assets/975c3129-bdc9-4386-83f4-489a02f0c2c9)
Since that offset is in world units, it also has to account for the scale of the volume.